paper.md

```diff
@@ -233,6 +233,18 @@ This allowed DataLad and many other projects to facilitate access to an ever-gro
 
 # Overview of the DataLad and its ecosystem
 
+## Design principles
+
+The development of DataLad is guided by a few fundamental principles to ensure its open and domain agnostic nature, to maximize the longterm utility of its datasets and to minimze the technical debt introduced by using it:
+
+- There are only two recognized entities: datasets and files
+- A dataset is a Git repository with an optional annex
+- Minimization of custom procedures and data structures: Users must not loose data or data access, if DataLad would vanish
+- Complete decentralization, no required central server or service
+
```

Comment:
I don't know if it is better discussed here, or in #65, but isn't "maximizing interoperability" also a design principle?

Reply:
Tend to agree, although it seems hard to me to convey what that means.

Reply:
Suggestion: How about keeping it 4 points, but add interoperability to "complete decentralization":

"Complete decentralization, with no required central server or service, but maximum interoperability to allow (re-)use of existing 3rd-party data resources and infrastructure"

Reply:
A bit long still, but I'll take it. :)